job-descriptions/product-manager.md

```diff
@@ -0,0 +1,74 @@
+- Our developers use the product, so they care deeply about product matters. You won't be fighting apathy.
+
+Review our [roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) to understand how our engineering team is organized and our [OKRs](https://about.sourcegraph.com/company/okrs) to understand the goals you will be helping drive.
+
+#### Responsibilities
+
+You will:
+
+- Own an area of the product, and create and drive its product roadmap.
+- Collaborate with engineers and designers to ideate, plan, and build the solutions to customer problems.
+- Use customer feedback to validate priorities, improve documentation, and define product requirements.
+- Be transparent in your internal and external communication. Sourcegraph is open source, and most discussion occurs in public, or in channels where you communicate directly with our customers.
+- Be a passionate advocate for a delightful user experience.
+
+#### Qualifications
+
+- Experience working as a software engineer.
+- Strong understanding of Git and Git workflows.
+- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
+- Ability to stay focused on goals and strategically prioritize work.
+- Deep passion for developer tools and productivity enhancements.
+- Demonstrated leadership, the ability to drive product strategy and work well with senior engineers and designers.
+
+#### Nice to haves
+
+- You have experience working as a product manager on a software development product.
+- You have experience building developer tools.
+- You have experience working with remote teams.
+
+### Location
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world.
+
+### Compensation and benefits
+
+We provide [competitive pay and equity](https://about.sourcegraph.com/handbook/people-ops/compensation) because we want you to act like a business owner and share in the success of Sourcegraph. We also provide [many benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.
+
+### Interview process
+
+1. You [apply here](<TODO(noemi)>).
+1. We set up a 30 minute call to chat with you about Sourcegraph to find out what you are looking for in your next role.
+1. We give you a small case study project to see how you would approach the job at Sourcegraph.
+1. We schedule 4 hours of remote interviews over video chat across multiple days.
+1. We check your references.
+1. We make you a job offer.
+
+We also expect you to be interviewing us too, so ask us any questions you have along the way!
```

Comment:
```suggestion
We expect you to be asking questions to ensure this is an environment that suits your working style and empowers you to do your best work
```

Reply:
I like this, updating a bit more. @nicksnyder to thumbs up on changing this for all job descriptions.

```suggestion
We expect you to be interviewing us too, so we look forward to any questions you have! We want to ensure this is an environment that suits your working style and empowers you to do your best work.
```

Reply:
I would invert it to put the motivation first.

```suggestion
We want to ensure Sourcegraph is an environment that suits your working style and empowers you to do your best work, so we are eager to answer any questions that you have about us at any point in the interview process.
```